src/main/index.js

```diff
@@ -429,8 +450,26 @@ app.on('before-quit', event => {
     // event will be triggered re-entrantly once preparation is done.
     event.preventDefault();
     shutdownDaemonAndQuit();
-  } else {
-    console.log('Quitting.');
+  } else if (autoUpdateDownloaded) {
+    if (autoUpdateAccepted) {
+      // User accepted the update, so install the update and restart.
+
+      autoUpdater.quitAndInstall();
+    } else if (process.platform == 'win32' && !showingAutoUpdateCloseAlert) {
+      // We have an update downloaded, but the user declined it (or closed the app without
+      // accepting it). Now the user is closing the app, so the new update will install.
+      // On Mac this is silent, but on Windows they get a confusing permission escalation
+      // dialog, so we show Windows users a warning dialog first.
+      event.preventDefault();
+      showingAutoUpdateCloseAlert = true;
+      dialog.showMessageBox({
+        title: "LBRY will upgrade",
+        message: "Please select \"Yes\" at the upgrade prompt shown after the app closes.",
```

Comment:
"LBRY has a pending upgrade. Please select "Yes" to install it on the prompt shown after this one."

(Assuming this is correct, and that prompt isn't shown until I ACK this one).

Reply:
You are correct, and I'll update the wording (along with all of the other wording changes you mentioned)